test/hotspot/jtreg/vmTestbase/nsk/jdb/caught_exception/caught_exception002/caught_exception002a.java

```diff
@@ -23,32 +23,29 @@
 
 package nsk.jdb.caught_exception.caught_exception002;
 
-import nsk.share.*;
-import nsk.share.jpda.*;
-import nsk.share.jdb.*;
+import nsk.share.Log;
+import nsk.share.jdb.JdbArgumentHandler;
 
-import java.io.*;
+import java.io.PrintStream;
 
-/* This is debuggee aplication */
+/* This is debuggee application */
 public class caught_exception002a {
 
-    /* TEST DEPENDANT VARIABLES AND CONSTANTS */
-    static final String PACKAGE_NAME = "nsk.jdb.caught_exception.caught_exception002";
-
-    public static void main(String args[]) {
-       caught_exception002a _caught_exception002a = new caught_exception002a();
-       System.exit(caught_exception002.JCK_STATUS_BASE + _caught_exception002a.runIt(args, System.out));
+    public static void main(String[] args) {
+        caught_exception002a _caught_exception002a = new caught_exception002a();
+        System.exit(caught_exception002.JCK_STATUS_BASE + _caught_exception002a.runIt(args, System.out));
     }
 
-    static void lastBreak () {}
+    static void lastBreak() {
+    }
 
-    public int runIt(String args[], PrintStream out) {
+    public int runIt(String[] args, PrintStream out) {
```

Comment:
Is there a style guide that says this is the preferred way to declare an array type? I count 3700 occurrences of "String args[]" in ours tests.

Reply:
yes, there is, e.g. [Java Style Guidelines](http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html#toc-variable-declarations) by Andreas Lundblad:
> Square brackets of arrays should be at the type (String[] args) and not on the variable (String args[]).

although @aioobe's guidelines have been accepted (yet?) to the openjdk guide (see openjdk/guide#14), this particular item is agreed on and accepted almost unanimously in the industry.

Reply:
ok